### PR TITLE
[MIG] project_task_category: Migration to 11.0

### PR DESCRIPTION
**Pull request by igallart**

### CODE REVIEW

File: project_task_category/__manifest__.py [hunk 1]
```diff
@@ -5,8 +5,9 @@
 {
     'name': 'Project Task Categories',
     'summary': 'Allow unique category for Tasks',
-    'version': '10.0.1.1.0',
+    'version': '11.0.1.0.0',
     'author': 'Elico Corp, Odoo Community Association (OCA)',
+    'contributors': "Punt Sistemes",
```
Comment (pedrobaeza) [inline project_task_category/__manifest__.py]: This should go on the Contributors section of the README, not here

File: project_task_category/models/project_category.py [hunk 2]
```diff
@@ -11,4 +11,6 @@ class ProjectCategory(models.Model):
     _name = 'project.category'
     _inherit = 'project.tags'
 
-    description = fields.Char()
+    description = fields.Char(
+        string='Category'
```
Comment (pedrobaeza) [inline project_task_category/models/project_category.py]: Comma at the end.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by devang-dreambits — COMMENTED** [review]
The module is Project Task Category and found the model name as 'project.category' (https://github.com/OCA/project/pull/393/files?utf8=%E2%9C%93&diff=unified#diff-6362ed1d7b90e33ee93bfc70a4fe2969R11)

Shall the model name be changed to 'task.category' to make it more appropriate or keep it for sake of backward compatibility?

File: project_task_category/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Project Task Categories',
+    'summary': 'Allow unique category for Tasks',
+    'version': '11.0.1.0.0',
+    'author': 'Elico Corp, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'category': 'Project Management',
+    'website': 'https://www.elico-corp.com',
```
Comment (nikul-serpentcs) [inline project_task_category/__manifest__.py]: @igallart IMHO change URL as Per [This](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10).

File: project_task_category/data/project_data.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline project_task_category/data/project_data.xml]: @igallart IMHO Directly use `<odoo noupdate="1">`

File: project_task_category/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
```
Comment (nikul-serpentcs) [inline project_task_category/models/__init__.py]: @igallart IMHO Remove copyright from all  __init__.py

File: project_task_category/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# © 2017 Elico Corp (https://www.elico-corp.com).
```
Comment (tarteo) [inline project_task_category/__manifest__.py:1]: Can you change the copyright sign to 'Copyright'

File: project_task_category/models/project_category.py [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
```
Comment (tarteo) [inline project_task_category/models/project_category.py:2]: Same here

File: project_task_category/models/project_category.py [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (tarteo) [inline project_task_category/models/project_category.py:1]: This is unnecessary

File: project_task_category/models/project_category.py [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProjectCategory(models.Model):
+    """Data Model for Project Category."""
+
+    _name = 'project.category'
```
Comment (tarteo) [inline project_task_category/models/project_category.py:11]: I think it's better to call this _name = 'project.task.category'

**Review by tarteo — CHANGES_REQUESTED** [review]
@igallart Are you gonna finish the migration?

Because I want to add some functionalities make some changes to this module:

- Make one module called project_category which makes it work for both projects and tasks
- Add fields parent_id and child_ids

**Comment by pedrobaeza** [comment]
We already added https://github.com/OCA/project/tree/11.0/project_category, so I think it's better to migrate and expand that one.

**Comment by tarteo** [comment]
@pedrobaeza Yes :+1:

**Comment by igallart** [comment]
If this pull request is not necessary, it could to be deleted?

**Pull request closed without merging**